scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -1330,6 +1330,8 @@ object Mod {
   class Transparent() extends Mod
   @ast
   class Erased() extends Mod
+  @ast
```

Comment:
either here, or in Dialect when you add an appropriate flag, could you please add a link to the documentation which describes this feature?

Reply:
Added it to the dialect flag comment